recipes/orsum/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.0.0" %}
+{% set sha256hash = "95ff71d6a9d4208f3e9e10470f4ea3eb15a99ad5a17110c93d1a635e47390783" %}
+package:
+  name: orsum
+  version: {{ version }}
+
+source:
+  url: https://github.com/ozanozisik/orsum/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256hash }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+  run:
+    - python >=3.6
+    - argparse
+    - numpy >=1.18.0
+
+test:
+  commands:
+    - orsum -h
```

Comment:
See if this works:
```suggestion
    - orsum.py -h
```

Reply:
You also may need to add a build script that makes `orsum.py` executable.

For example, in a `build.sh` file, you could have:
```bash
#!/bin/bash

cp *.py $PREFIX/bin
chmod +x $PREFIX/bin/orsum.py
```

Reply:
@fanninpm Thank you very much, - orsum.py -h and creating build.sh solved it.